src/test/java/com/uber/cadence/activity/ActivityOptionsTest.java

```diff
@@ -57,7 +56,7 @@ public void testOnlyOptionsPresent() throws NoSuchMethodException {
     Assert.assertEquals(o, ActivityOptions.merge(a, null, o));
   }
 
-  @MethodRetry(initialIntervalSeconds = 3)
+  @MethodRetry(initialIntervalSeconds = 3, expirationSeconds = 100)
```

Comment:
What's the difference between this and the test above? We are not using this annotation anyway.